tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/GraphQL/HotChocolate/ExecuteAsyncIntegrationExtraV14.cs

```diff
@@ -0,0 +1,47 @@
+// <copyright file="ExecuteAsyncIntegrationExtraV14.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.ComponentModel;
+using Datadog.Trace.ClrProfiler.CallTarget;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.GraphQL.HotChocolate
+{
+    /// <summary>
+    /// HotChocolate.Execution.Processing.WorkScheduler calltarget instrumentation to retrieve OperationType
+    /// </summary>
+    [InstrumentMethod(
+        IntegrationName = HotChocolateCommon.IntegrationName,
+        MethodName = "ExecuteAsync",
+        ReturnTypeName = "System.Threading.Tasks.Task`1[HotChocolate.Execution.IOperationResult]",
+        ParameterTypeNames = new[] { "HotChocolate.Execution.Processing.OperationContext" },
+        AssemblyName = "HotChocolate.Execution",
+        TypeName = "HotChocolate.Execution.Processing.QueryExecutor",
+        MinimumVersion = "14",
+        MaximumVersion = "14.*.*")]
+    [Browsable(false)]
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class ExecuteAsyncIntegrationExtraV14
+    {
+        /// <summary>
+        /// OnMethodBegin callback
+        /// </summary>
+        /// <typeparam name="TTarget">Type of the target</typeparam>
+        /// <typeparam name="TOperationContext">Type of the first parameter</typeparam>
+        /// <param name="instance">Instance value, aka `this` of the instrumented method.</param>
+        /// <param name="operationContext">Operation context</param>
+        /// <returns>Calltarget state value</returns>
```

Comment:
those kind of "auto-generated" comments don't add value, and just make the codebase more verbose for no added benefit, it's better to add no comment in this case.
(same remark for other instances of this below)

Reply:
Agreed. I just copied what was there in the old code, since I thought that was already according to your guidelines :) Removed it again.